2021-07-27-minisummit-agenda.md

```diff
@@ -0,0 +1,127 @@
+[2020](https://3mdeb.com/events/#Qubes-OS-and-3mdeb-minisummit2020).) This
+year's event will take place across two virtual sessions on August 3 and 10.
+Each day, there will be four talks, intermixed with Q&A time. An abstract for
+each talk is provided below.  The discussion section will be a live meeting on
+Jitsi, with details to follow.  The whole event will be also streamed on
+[3mdeb's YouTube
+channel](https://www.youtube.com/channel/UC_djHbyjuJvhVjfT18nyqmQ), where we
+will also accept questions. We invite everyone interested to join!
+
+## Agenda for August 3
+
+| Time (UTC)     | Event description
+| -------------- | ----------------- 
+| 18:00 -- 18:15 | Welcome and introduction by Piotr Król
+| 18:15 -- 19:00 | "Qubes OS 4.1 highlights" by Marek Marczykowski-Górecki
+| 19:00 -- 19:45 | "First Impressions Count: Onboarding Qubes Users Through an Integrated Tutorial" by deeplow
+| 19:45 -- 20:15 | Break
+| 20:15 -- 21:00 | "Wyng-backups: revertible local and remote known safe Qubes OS states (including dom0)" by Thierry Laurion
+| 21:00 -- 21:45 | "SRTM and Secure Boot for VMs" by Piotr Król
+| 21:45          | vPub, informal afterparty
+
+## Agenda for August 10
+
+| Time (UTC)     | Event description
+| -------------- | ----------------- 
+| 18:00 -- 18:15 | Welcome and introduction by Piotr Król
+| 18:15 -- 19:00 | "Usability Within A Reasonably Secure, Multi-Environment System" by Nina Alter
+| 19:00 -- 19:45 | "Qubes OS Native App Menu: UX Design and Implementation" by Marta Marczykowska-Górecka and Nina Alter
+| 19:45 -- 20:15 | Break
+| 20:15 -- 21:00 | "A brief history of USB camera support in Qubes OS" by Piotr Król
+| 21:00 -- 21:45 | "How to setup BTC and XMR cold storage in Qubes OS" by Piotr Król
+| 21:45          | vPub, informal afterparty
+
+## Abstracts of the talks
+
+### "Qubes OS 4.1 highlights" by Marek Marczykowski-Górecki
+
+The upcoming Qubes OS 4.1 release is full of new exciting features, ranging
+from a technology preview of the GUI domain to subtle, yet important, Qrexec
+improvements. In this talk I will give a brief overview of them and demo a
+select few.
+
+### "First Impressions Count: Onboarding Qubes Users Through an Integrated Tutorial" by deeplow
+
+We may all relate to having a rough time when starting using Qubes --- be that
+because we're coming from Windows and everything is different or because we
+come from Linux and many things don't work like we expect them to. Apart from
+the usual challenges of going into a different system, Qubes has the additional
+one of requiring a fundamentally different way of thinking about your computer
+(a hypervisor mental-model). Smoothing out this transition is particularly
+important as Qubes aims to target vulnerable populations that are less
+technically inclined and have less time to explore and read the documentation.
+
+The solution proposed by deeplow is to implement an integrated onboarding
```

Comment:
We shouldn't link to only one person's name. If we're going to link to anyone's name, we should link to everyone's name. Can go either way; up to you.

Reply:
If we do, probably best to do it in the tables above.

Reply:
I'm fine with dropping (I copied it from the original abstract we've got).